server/index.js

```diff
@@ -0,0 +1,23 @@
+const thoughtsRoutes = require('./routes/thought')
+const express = require('express');
+const cors = require('cors');
+const server = express();
+const bodyParser = require("body-parser")
+server.use(cors());
+server.use(express.json());
+server.use(bodyParser.json())
+const { Pool } = require("pg");
+const pool = new Pool()
```

Comment:
This is not needed in this file. I am not sure myself what exactly pool does but my guess is that it 'brings' the postgres database into the file you are working on. Since you are not interacting with postgres database in this file, you do not need to add this here. Instead, you would require this in your models file since this is where you'll be selecting, creating, updating and deleting in your database.